docs/contributors/principles.md

```diff
@@ -0,0 +1,33 @@
+# Principles of Minikube
+
+The primary goal of minikube is to make it simple to run Kubernetes on your local machine, both for getting started and day-to-day development workflows.
```

Comment:
Should specify what "on your local machine" means - or maybe just drop it and say "locally" (vague on purpose).

Many people try the `none` driver only to be disappointed, since it is not intended to run by the end-user at all...